tools/zbrealign

```diff
@@ -15,7 +15,7 @@ from killerbee import *
 try:
 	from scapy.all import *
 except ImportError:
-	print 'This tool requires Scapy to be installed (specifically scapy-com edition).'
+	print('This tool requires Scapy to be installed (specifically scapy-com edition).')
```

Comment:
Ditto below warning can be changed -- see https://github.com/secdev/scapy/tree/master/scapy/layers that both layers are now in trunk.